src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Azs.Fabric.Admin.psd1

```diff
@@ -17,7 +17,7 @@ Licensed under the MIT License. See License.txt in the project root for license
     RootModule        = 'Azs.Fabric.Admin.psm1'
 
     # Version number of this module.
-    ModuleVersion     = '0.2.1'
+    ModuleVersion     = '0.4.0
```

Comment:
This should be 0.3.0, why did you skip?

Reply:
Fixed, I think we need to match the PS version with SDK version before.